web3rpc/rpc-specs/code-samples/java/src/main/java/opensdk/sdk/apis/eth/block/EthGetBlockReceiptsExample.java

```diff
@@ -0,0 +1,20 @@
+package opensdk.sdk.apis.klay.block;
+
+
+import opensdk.sdk.apis.constant.UrlConstants;
```

Comment:
I don't want to expose the 'opensdk' in the code-samples. Could you refactor this?